src/pages/BuildPage/BuildPage.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Redirect, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import './BuildPage.css';
+
+class BuildPage extends Component {
+  render() {
+    const { selectedBuild } = this.props;
+    if (!selectedBuild) {
```

Comment:
if there's no selected build (in case of refresh, or manually typing an url, this build should still be rendered - by getting specific build from API before render, using the info from url

Reply:
(I just realized I didn't explicitly write this in task, sorry - but still do it if you have time)